package-lock.json

```diff
@@ -32860,7 +32863,9 @@
       }
     },
     "http-proxy-middleware": {
-      "version": "2.0.3",
+      "version": "2.0.7",
+      "resolved": "https://registry.npmjs.org/http-proxy-middleware/-/http-proxy-middleware-2.0.7.tgz",
```

Comment:
Please avoid changes to the package lock, this update may be useful but should be a standalone PR. 

In the future it's best to use `npm ci` instead of `npm install` when running locally. This way the package lock file won't change. 

See https://docs.wagtail.org/en/stable/contributing/developing.html#setting-up-the-wagtail-codebase